cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/ProvisionerStateTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class ProvisionerStateTest {
+  @Test
+  public void testPopulateProvisionerState() {
+    ProvisionerState.State expectedState = ProvisionerState.State.COMPLETED;
+    String expectedSummary = "Test summary.";
+
+    ProvisionerState provisionerStateResults = new ProvisionerState(expectedState, expectedSummary);
+
+    assertEquals(expectedState, provisionerStateResults.state());
+    assertEquals(expectedSummary, provisionerStateResults.summary());
+    assertEquals(provisionerStateResults.createdMs(), provisionerStateResults.updatedMs());
+  }
+
+  @Test
+  public void testProvisionerUpdateState() {
```

Comment:
Looks like this test covers almost everything in the `testPopulateProvisionerState`. Can we avoid this duplication and drop the other repetitive test?